cg/store/crud/update.py

```diff
@@ -106,3 +110,11 @@ def update_analysis_upload_started_at(
         analysis = self.get_analysis_by_entry_id(analysis_id)
         analysis.upload_started_at = upload_started_at
         self.session.commit()
+
+    def update_order_type_applications(self, application: Application, form: Form):
+        """Updates the order_type_application table based on the updated Application object"""
+        self._get_query(OrderTypeApplication).filter_by(application_id=application.id).delete()
+        selected_order_types: list[OrderType] = form["suitable_order_types"].data
+        for order_type in selected_order_types:
```

Comment:
and then you can call `store.link_order_types_to_application` (not yet merged)

Reply:
done